nano/node/transport/tcp.cpp

```diff
@@ -10,6 +10,13 @@ nano::transport::channel_tcp::channel_tcp (nano::node & node_a, std::weak_ptr<na
 {
 }
 
+nano::transport::channel_tcp::channel_tcp (nano::node & node_a, nano::tcp_endpoint const & fake_endpoint_a) :
```

Comment:
I am confused by what is happening here.
How is this channel getting to connected status?
I fear we are adding complexity to something that is already too complex.

Reply:
Agreed. Using an InProc channel would solve this too

Reply:
What we really need is a null/dummy channel that doesn;t connect to anything and simply allows for its attributes to be manipulated.